vllm/model_executor/layers/quantization/mxfp4.py

```diff
@@ -132,12 +132,15 @@ def get_mxfp4_backend(with_lora_support: bool) -> Mxfp4Backend:
             )
 
         # If FlashInfer is not available, try either Marlin or Triton
-        if (
-            envs.VLLM_MXFP4_USE_MARLIN
-            or current_platform.get_device_capability()[0] < 9
-            or not has_triton_kernels()
-            or not is_torch_equal_or_newer("2.8.0")
-        ):
+        triton_kernels_supported = (
+            has_triton_kernels()
+            and is_torch_equal_or_newer("2.8.0")
```

Comment:
I think vllm requires torch 2.9.0 now, could we remove this?

Reply:
CUDA does, but other hardware does not. Even though this is in a CUDA branch, I think some people still build with their own versions of torch. So would like to keep for now